catalog-info.yaml

```diff
@@ -36,6 +39,46 @@ spec:
         SLACK_NOTIFICATIONS_ALL_BRANCHES: "false"
         SLACK_NOTIFICATIONS_ON_SUCCESS: "false"
 
+---
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: buildkite-pipeline-docs-pr
+  description: Build the docs pull requests
```

Comment:
```suggestion
  description: Build the docs on pull requests to the `elastic/docs` repo.
```

Reply:
I think the description of the `buildkite-pipeline-docs` pipeline could also be changed from "Build the Docs" to something like "Build and Publish the docs"